### ISSUE TITLE
Automatic update check

### ISSUE THREAD

**Issue by lukasbestle** [getkirby/kirby#2820]
See #2633 and #2651.

**Comment by distantnative** [comment]
https://kirby.nolt.io/111

**Comment by lukasbestle** [comment]
@distantnative To be honest I think this is something we should include in 3.5.0 given that the feature is mostly implemented already.

The code we have written will become more and more out of date the longer we wait and we will have forgotten in a few months how our code works. So it would require much more time to implement it later.

Also the point of the update check is to notify as many users as possible about security updates. The earlier we introduce the check, the more sites will run it.

What do you think? Also pinging @bastianallgeier for discussion.

**Comment by distantnative** [comment]
@lukasbestle since we had to leave the `storybook` episode behind got the moment, this won’t easily make it into 3.5 - the whole UI part needs to be rewritten and many parts the UI integration relied on (e.g. alerts) aren’t available.

Since we rather want to keep releases focused and made the builder the priority for 3.5, I don’t see this as part of 3.5 as well.

I know it’s a pity to leave so much code on the side, but it’s a lesson we had to learn with the whole storybook code. We will hopefully be able to pick up piece by piece again and integrate them in future versions - but not as the grand scheme we thought we could.
